README.template.md

```diff
@@ -59,4 +59,12 @@ Coming soon!
 
 Coming soon!
 
+### Running Tests
+
+```bash
+export DOCKER_COMMAND="php vendor/phpunit/phpunit/phpunit --configuration phpunit.xml"
+docker build --tag php-test --build-arg token=<YOUR_AUTH_TOKEN> .
+docker run -d -it php-test bash -c $DOCKER_COMMAND
```

Comment:
This didn't work for me; to get it to work I ran this command:

```
docker run -it php-test php vendor/phpunit/phpunit/phpunit --configuration phpunit.xml
```

When you follow my advice above about making the build more generic, you are going to end up with some additional args that are required for the docker run command; e.g. some `-e`s to set env vars, and probably a `-v` to mount the sdk code.  At that point the `docker run` command will be fairly long so it will probably be a good idea to have some bash scripts that people can use.  e.g. `dev-docker-build.sh`, and `dev-run-integration-tests.sh` or something like that.